resource_management/azure_arm.gemspec

```diff
@@ -0,0 +1,40 @@
+# encoding: utf-8
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+
+version = File.read(File.expand_path('../ARM_VERSION', __FILE__)).strip
+
+Gem::Specification.new do |spec|
+  spec.name          = 'azure_arm'
+  spec.version       = version
+  spec.authors       = 'Microsoft Corporation'
+  spec.email         = 'azrubyteam@microsoft.com'
+  spec.description   = 'Microsoft Azure Web Apps Management Client Library for Ruby'
+  spec.summary       = 'Official Ruby client library to consume Microsoft Azure Web Apps Management services.'
+  spec.homepage      = 'http://github.com/azure/azure-sdk-for-ruby'
+  spec.license       = 'MIT'
+
+  spec.required_ruby_version = '>= 1.9.3'
+
+  spec.add_development_dependency 'bundler', '~> 1.9'
+  spec.add_development_dependency 'rake', '~> 10'
+  spec.add_development_dependency 'rspec', '~> 3'
+  spec.add_development_dependency 'dotenv', '~> 2'
+
+  spec.add_runtime_dependency 'azure_mgmt_authorization', version
```

Comment:
should we be adding the dependency to a specific version or use the pessimistic operator?